### DISCUSSION TITLE
邮箱通知

### DISCUSSION THREAD

**Discussion by Nofated095** [walinejs/waline#747] [Q&A] [answered]
我按照文档里面的内容在 vercel 添加了邮箱的函数来使用验证码和邮件回复功能。但是在redeploy后无法注册账户并且也收不到邮件。

使用的是 Microsoft Outlook 365，源自MS E5

**Comment by Nofated095** [comment]
> 我按照文档里面的内容在 vercel 添加了邮箱的函数来使用验证码和邮件回复功能。但是在redeploy后无法注册账户并且也收不到邮件。
> 
> 使用的是 Microsoft Outlook 365，源自MS E5

https://comment-ivgt.vercel.app/

**Comment by lizheming** [comment]
后台日志可以看具体的报错信息，同时建议换其它邮箱试试。相关 issue https://github.com/walinejs/waline/issues/483

**Comment by Nofated095** [comment, marked as answer]
更换到126后正常。

但是一般情况下 365 的邮件不会被识别成垃圾邮件，126经常会（
